src/content-scanner/readScannerUrlConfig.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { logger } from "matrix-js-sdk/src/logger";
+import SdkConfig from "matrix-react-sdk/src/SdkConfig";
+
+import { ContentScannerConfig } from "./ContentScannerConfig";
+
+/**
+ * Reads "content_scanner.url" from the config.
+ * Logs a warning if it is not present.
+ */
+export const readScannerUrlConfig = (): string => {
+    const contentScannerUrl = (SdkConfig.get() as unknown as ContentScannerConfig)?.content_scanner?.url;
```

Comment:
On Tchap the content scanner is the base as the server. We have 17 differents homeservers different